alf/algorithms/config.py

```diff
@@ -126,6 +132,25 @@ def __init__(self,
                 is an on-policy sub-algorithm, we can enable this flag for its
                 training. ``OnPolicyAlgorithm`` always unrolls with grads and this
                 flag doesn't apply to it.
+            async_unroll: whether to unroll asynchronously. If True, unroll will
+                be performed in parallel with training.
+            max_unroll_length: the maximal length of unroll results for each iteration.
+                If the time for one step of training is less than the time for
+                unrolling ``max_unroll_length`` steps, the length of the unroll
+                results will be less than ``max_unroll_length``. Only used if
+                ``async_unroll`` is True and unroll_length==0.
+            unroll_queue_size: the size of the queue for transmitting unroll
+                results from the unroll process to the main process. Only used
+                if ``async_unroll`` is True. If the queue is full, the unroll process
+                will wait for the main process to retrieve unroll results from
+                the queue before performing more unrolls.
+            unroll_step_interval: if not zero, the time interval in second
```

Comment:
This interval can only lower bound the time interval between two env steps? If the rollout step is really slow, then the actual interval might be longer than this one?

Reply:
You are right. This is only useful if env step and rollout step together spend less than unroll_step_interval.